### DISCUSSION TITLE
ToIP Interop

### DISCUSSION THREAD

**Discussion by vongohren** [decentralized-identity/interoperability#60] [Topics]
I want to hear more about the work of interop happening in ToIP.
Not sure how to go about it, but maybe someone here knows?

**Comment by G-T8** [comment]
Hi Snow,
Yes, I too am curious about the next steps. Interops with ToIP looks streamline, with a great team of communicators, and excellent docs. If you hear anything, please lmk and I'll do the same in return. Best,
